src/main/java/com/targomo/client/api/request/PointOfInterestRequest.java

```diff
@@ -46,6 +48,10 @@ public PointOfInterestRequest(TravelOptions travelOptions) {
 		this.travelOptions = travelOptions;
 		this.headers = new MultivaluedHashMap<>();
 	}
+	public PointOfInterestRequest(TravelOptions travelOptions, boolean forceRecalculate) {
```

Comment:
forceRecalculate and cacheResult are already present in TravelOptions. pls them rather that extending the constructors